netbout-web/src/main/java/com/netbout/dynamo/DyBase.java

```diff
@@ -61,6 +61,24 @@
      * Public ctor.
      */
     public DyBase() {
+        this.region = DyBase.initialize();
+    }
+
+    @Override
+    public User user(final URN urn) {
+        return new DyUser(this.region, urn);
+    }
+
+    @Override
+    public void close() throws IOException {
+        // nothing to do here
+    }
+
+    /**
+     * Initialize a region instance.
+     * @return The initialized region instance
+     */
+    private static Region initialize() {
```

Comment:
@danff please read http://www.yegor256.com/images/books/elegant-objects/seven-pages.pdf - this method should be named like a noun